FrontEnd/src/components/ProfileDetail/DetailedInfo/PhoneEmail.jsx

```diff
@@ -4,11 +4,25 @@ import useSWR from 'swr';
 import { useAuth } from '../../../hooks';
 import { PropTypes } from 'prop-types';
 import classes from './PhoneEmail.module.css';
+import { CheckOutlined, CopyOutlined} from '@ant-design/icons';
 
 function PhoneEmail ({ profileId, personId }) {
     const [isContactsShown, setContactsShown] = useState(false);
+    const [isPhoneCopied, setIsPhoneCopied] = useState(false);
+    const [isEmailCopied, setIsEmailCopied] = useState(false);
     const { user } = useAuth();
 
+    const copyContent = key => {
+      navigator.clipboard.writeText(profileData[key]);
```

Comment:
What happens if error?

Reply:
Yes, I agree, the only question is how to handle it. Should we show some modal with error message? I think we need to discuss it and this is related to broader topic of error visual handling.

Reply:
Sometimes we just use 'catch (error) {
      console.error(error);
    }' 
to see error in the console. If later we'll get a desine we'll change it